### ISSUE TITLE
It is a virus.

### ISSUE THREAD

**Issue by Corionis** [tokyoneon/B2E#11]
Windows 11 will not run the EXE built by this virus-injecting trojan.

**Comment by eebssk1** [comment]
That's Google Golang's problem. It has been fixed. But this program uses part of it which has never been updated.

**Comment by cody151** [comment]
I don't think so buddy, it is simply supposed to convert from batch to an exe. It is clearly injecting trojan malware into the final exe. Avoid this like the plague.

**Comment by eebssk1** [comment]
> I don't think so buddy, it is simply supposed to convert from batch to an exe. It is clearly injecting trojan malware into the final exe. Avoid this like the plague.

It's based on (really old)go.
